src/Microsoft.DotNet.Darc/src/Darc/Options/AddSubscriptionCommandLineOptions.cs

```diff
@@ -52,6 +52,12 @@ class AddSubscriptionCommandLineOptions : CommandLineOptions
         [Option("read-stdin", HelpText = "Interactive mode style (YAML), but read input from stdin. Implies -q")]
         public bool ReadStandardIn { get; set; }
 
+        [Option("trigger", SetName = "triggering", HelpText = "Automatically trigger the subscription on creation.")]
```

Comment:
Doesn't an default option makes sense here and so we'll only need one of these options?

Reply:
Having the same SetName means that the two options are part of a "set" and they should be passed together. I don't think that's what we want for these two. I think we want them to have different `SetName` so they are mutually exclusive? (like we do with the `exact` and `regex` options for the subscription filters? https://github.com/dotnet/arcade-services/blob/d11409116da97e2ae9ff73476dce00c23015886b/src/Microsoft.DotNet.Darc/src/Darc/Options/SubscriptionsCommandLineOptions.cs#L37-L41